### ISSUE TITLE
reset import does not work on google chrome

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ravi5677/gisgraphy#25]
```
go to /admin/resetimport.html
confirm and reset
the reset is not done

What is the expected output? What do you see instead?
the reset should be done



```

Original issue reported on code.google.com by `davidmasclet@gmail.com` on 21 Jul 2012 at 12:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `davidmasclet@gmail.com` on 31 Jan 2014 at 9:11
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I verified the same on Chrome on Linux. Never starts the reset but shows 
"The reset is in progress, please wait. See log file to see the status."

The logs show no progress

Tried the same on the same server with Firefox Linux and it finished in time 
and the log showed lots of activity.

I'm running Version 3.0-beta2
```

Original comment by `carlos.t...@globant.com` on 6 Aug 2014 at 2:10

**Comment by GoogleCodeExporter** [comment]
```
fixed in 4.0.
```

Original comment by `davidmasclet@gmail.com` on 6 Aug 2014 at 6:57
